tempesta_fw/sock_srv.c

```diff
@@ -842,6 +842,19 @@ static struct {
 	bool sched		: 1;
 } __attribute__((packed)) tfw_cfg_is_set;
 
+typedef struct {
+	struct list_head	list;
+	char			*name;
+	TfwServer		*srv;
+	unsigned int		flags;
+} TfwSrvHealth;
+
+/* Types of configuration for server health monitor. */
+#define TFW_SRV_HEALTH_ENABLE		0x0001
+#define TFW_SRV_HEALTH_DISABLE		0x0002
+
+static struct list_head tfw_cfg_health_lst = LIST_HEAD_INIT(tfw_cfg_health_lst);
```

Comment:
```C
static LIST_HEAD(tfw_cfg_health_lst);
```

Reply:
This code has been redesigned (removed `tfw_cfg_health_lst`).